backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewEdgeService.java

```diff
@@ -182,4 +200,63 @@ public boolean checkTransitionEdgeTarget(Element source, Element target) {
         }
         return false;
     }
+
+    /**
+     * Succession edge label getter service.
+     *
+     * @param succession
+     *            The succession edge
+     * @return the label of the given {@link Succession}
+     */
+    public String getSuccessionLabel(Succession succession) {
+        String resultLabel = "";
```

Comment:
Please prefer a StringBuilder

Reply:
Done

Reply:
I wanted a StringBuilder not a StringBuffer. I will rebase the PR change it myself.